openfasoc/MLoptimization/model.py

```diff
@@ -0,0 +1,47 @@
+#training import
+import gym
+import ray
+import ray.tune as tune
+from ray.rllib.algorithms.ppo import PPO
+from run_training import Envir
+from sky130_nist_tapeout import single_build_and_simulation
+sky130_nist_tapeout.path.append('../generators/gdsfactory-gen/')
+
+import argparse
+#
+#training set up
+parser = argparse.ArgumentParser()
```

Comment:
Refactor file to if name main format and all argparse's into a function called parse_args